cmd/govim/main.go

```diff
@@ -91,7 +91,8 @@ func launch(goplspath string, in io.ReadCloser, out io.WriteCloser) error {
 	var logFile *os.File
 	var writers []io.Writer
 
-	if getEnvVal(d.goplsEnv, config.EnvLog, "on") == "on" {
+	logging := csvToMap(getEnvVal(d.goplsEnv, config.EnvLog, "on"))
```

Comment:
We should build this map once and set it on the `govimplugin`. Which would also accurate reflect the semantics that we do not mean to support the semantics of the values changing.